src/components/MultiHopTrade/components/TradeInput/TradeInput.tsx

```diff
@@ -159,20 +165,34 @@ export const TradeInput = ({
     isLoading: isWalletReceiveAddressLoading,
   } = useTradeReceiveAddress()
 
+  const hasQuotes =
+    availableTradeQuotesDisplayCache.length > 0 || unavailableTradeQuotesDisplayCache.length > 0
+
   const isLoading = useMemo(
     () =>
       // No account meta loaded for that chain
-      Boolean(walletId && !isAnyGetAccountPortfolioLoadedForChainId) ||
-      (!shouldShowTradeQuoteOrAwaitInput && !isTradeQuoteRequestAborted) ||
+      // Note we're only concerned about this if the wallet supports the sell asset chain
+      // If it does not, users should still be able to see rates, though obviously won't be able to action them
+      // until connecting a wallet which does
+      Boolean(
+        walletId &&
+          walletSupportsSellAssetChain &&
+          !isAnyGetAccountPortfolioLoadedForSellAssetChainId,
+      ) ||
+      (!hasQuotes && isBatchTradeRateQueryLoading) ||
+      (!hasUserEnteredAmount && !isTradeQuoteRequestAborted) ||
```

Comment:
`shouldShowTradeQuoteOrAwaitInput` won't cut it here as it will wrongly evaluate to true because of  this guy:
https://github.com/shapeshift/web/blob/5b6d7d6bd9306b99755fade248e9a0ee50a42445/src/state/slices/tradeQuoteSlice/selectors.ts#L644 

- Kept only the `selectHasUserEnteredAmount` part 
- `selectIsAnySwapperQuoteAvailable` effectively handled by the `hasQuotes` above